docs/deploy.rst

```diff
@@ -1,7 +1,7 @@
 .. _deploy:
 
-Deploy
-======
+Deployment Guide
+================
```

Comment:
If we are going to enumerate every platform we can deploy on, it definitely shouldn't include instructions for each one. I'd rather just say that the system can be deployed on a wide variety of systems, from bare metal to IaaS to managed services like EBeanStalk.